website/content/api-docs/secret/identity/tokens.mdx

```diff
@@ -477,19 +515,26 @@ $ curl \
 ```json
 {
   "issuer": "https://example.com/v1/identity/oidc/plugins",
-  "authorization_endpoint": "",
-  "token_endpoint": "",
   "jwks_uri": "https://example.com/v1/identity/oidc/plugins/.well-known/keys",
-  "response_types_supported": null,
-  "subject_types_supported": ["public"],
-  "id_token_signing_alg_values_supported": ["RS256"],
-  "scopes_supported": null,
-  "token_endpoint_auth_methods_supported": null,
-  "claims_supported": null
+  "response_types_supported": [
+    "id_token"
+  ],
+  "subject_types_supported": [
+    "public"
+  ],
+  "id_token_signing_alg_values_supported": [
+    "RS256",
+    "RS384",
+    "RS512",
+    "ES256",
+    "ES384",
+    "ES512",
+    "EdDSA"
+  ]
 }
 ```
 
-## Read active public keys
+## Read plugin identity token issuer's public JWKS <EnterpriseAlert product="vault" inline />
```

Comment:
```suggestion
## Read the public JWKS  from a plugin identity token<EnterpriseAlert product="vault" inline />
```

Reply:
Similar to my other comment, this isn't technically correct. The "plugin identity token issuer" has a JWKS. Happy to apply a different suggestion!

Reply:
Tweaked suggestion